### PR TITLE
fix(*): Move conf annotation processor to kork-core

### PR DESCRIPTION
**Pull request by robzienert**

### CODE REVIEW

**Comment by jonsie** [comment]
That should do it.

File: kork-core/kork-core.gradle [hunk 1]
```diff
@@ -5,7 +5,9 @@ apply from: "$rootDir/gradle/lombok.gradle"
 dependencies {
   api(platform(project(":spinnaker-dependencies")))
   implementation project(":kork-exceptions")
+  annotationProcessor(platform(project(":spinnaker-dependencies")))
```
Comment (cfieber) [inline kork-core/kork-core.gradle]: I suspect we want this in the top level build.gradle on all subprojects

Comment (jonsie) [reply kork-core/kork-core.gradle]: I believe most of the services are already configured this way, for example:

https://github.com/spinnaker/orca/blob/master/build.gradle#L63

Comment (jonsie) [reply kork-core/kork-core.gradle]: But maybe you're referring to just kork - that was my initial inclination too.  Not sure if just `kork-core` will cover everything.

Comment (cfieber) [reply kork-core/kork-core.gradle]: sorry I actually meant the actual annotationProcessor dependency on spring-boot-configuration-processor could land at the top level on all subprojects. The annotationProcessor platform dependency is probably already there maybe for lombok

**Pull request closed without merging**